CHANGES.md

```diff
@@ -34,3 +35,19 @@
   - `subscription.getTopicAsTopicNameOneof()`: use `TopicNames.parse(subscription.getTopic())`
   - `subscription.getNameAsSubscriptionName()`: use `ProjectSubscriptionName.parse(subscription.getName())`
   - `snapshot.getNameAsSnapshotName()`: use `ProjectSnapshotName.parse(snapshot.getName())`
+
+# v0.44.0
```

Comment:
What happened to v0.37.0 and so on? o.O

Reply:
The smaller notes are written [here](https://github.com/GoogleCloudPlatform/google-cloud-java/releases). This file is usually for notes with significant explanations, etc.